storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -350,10 +350,21 @@ func getPromExemplars[T exemplarType](ctx context.Context, everyN *everyNTimes,
 		exemplar := pt.Exemplars().At(i)
 		exemplarRunes := 0
 
-		promExemplar := prompb.Exemplar{
-			Value:     exemplar.DoubleValue(),
-			Timestamp: timestamp.FromTime(exemplar.Timestamp().AsTime()),
+		var promExemplar prompb.Exemplar
```

Comment:
nit: I'd have initialized the exemplar with the timestamp only and then set its value in a switch instead of an if/else